langchain4j-ollama/src/main/java/dev/langchain4j/model/ollama/OllamaModel.java

```diff
@@ -26,11 +26,13 @@ public class OllamaModel {
     OllamaModel() {
     }
 
-    OllamaModel(String name, long size, String digest, OllamaModelDetails details) {
+    OllamaModel(String name, long size, String digest, OllamaModelDetails details, OffsetDateTime modifiedAt, String model) {
         this.name = name;
         this.size = size;
         this.digest = digest;
         this.details = details;
+        this.modifiedAt = modifiedAt;
+        this.model = model;
     }
 
     static Builder builder() {
```

Comment:
Should this method also be public?

Reply:
Looks like in `OllamaModels.modelCard` users can construct`OllamaModel` instance and pass it as param. So I think `OllamaModel` and `OllamaModelDetails` should have public access. WDYT?

Reply:
Ah yes, I forgot about that. When I proposed this method, my use case scenario was to:

1. Invoke `OllamaModels.availableModels`.
2. Pass the returned item list to `OllamaModels.modelCard`.

I never intended to instantiate the `OllamaModel` class myself and pass it to `OllamaModels.modelCard`. This method uses `OllamaModel.getName` and passes it to the other method variant, `OllamaModels.modelCard(String ollamaModelName)`. 

So, I’m not sure if the user should need to create a valid `OllamaModel` object with `OllamaModelDetails` when only the `name` is required.

However, if you see this as a valid use case, I am open to your suggestion.

Reply:
Hmm...Your use case scenario sounds reasonable, and it really doesn't require the user to manually construct the `OllamaModel`. But since we're exposing these methods(`deleteModel` and `modelCard` with `OllamaModel` param), users may have potential to use it anyway. So I think we should public them :)

Reply:
Now I see the updated comment. From my perspective, the first three classes:

- `OllamaModel`
- `OllamaModelDetails`
- `OllamaModelCard`

are DTOs created and provided by the API through the `OllamaModels` class. They were never intended to be created by user code. However, `OllamaModels` is different; it is more like a service class, so the user should be able to create and configure it as needed.

And now I see another comment :) As I mentioned previously, I am open to your suggestion.

Reply:
Ok, I pushed a new commit.

Reply:
Thank you!